distro.json

```diff
@@ -261,8 +261,8 @@
         },
         "linux": {
           "universal": {
-            "url": "https://nxxm.indigenious.io/distro/all/seafile_client/tipi_seafile_client_8.0.3p2_linux_amd64.zip",
-            "sha1": "9b0a5c87058b68cd466a3cc8e8cfd7d9cbd89df2",
+            "url": "https://nxxm.indigenious.io/distro/all/seafile_client/tipi_seafile_client_8.0.3p2_linux_amd64_16_04_v3.zip",
```

Comment:
Could be public from our build workflow and gh releases (instead from the distro file server)

Reply:
I created an issue for this in the sprint